kotlin-android/app/src/main/java/com/mongodb/tasktracker/MemberActivity.kt

```diff
@@ -0,0 +1,116 @@
+package com.mongodb.tasktracker
+
+import android.app.AlertDialog
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.EditText
+import android.widget.Toast
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import com.mongodb.tasktracker.model.*
+import io.realm.Realm
+import io.realm.mongodb.functions.Functions
+import org.bson.Document
+import java.util.*
+
+class MemberActivity : AppCompatActivity() {
```

Comment:
nit: Could we add a comment that gives a quick overview of what this Activity does?

Reply:
+1, going to add this for all activities (original implementation had these, but i haven't written all of the new descriptions yet)